### ISSUE TITLE
sesman login failed

### ISSUE THREAD

**Issue by Hexdump74** [neutrinolabs/xrdp#290]
Hi,

I downloaded yesterday (20th oct 2015) the latest version of xrdp-devel, and encountered a problem with sesman and PAM on my centos 6.6.
The /etc/pam.d/xrdp-sesman file created by the installation wasn't working and thus I got everytime a failed login.
I rewrite the file with :

> # %PAM-1.0
> 
> auth    required        pam_unix.so shadow nullok
> auth    required        pam_env.so readenv=1
> account required        pam_unix.so

and now the login are working.
I though you wanted to know.

Regards,
Emmanuel

**Comment by viewizard** [comment]
Gentoo, for example, use this one all the time:
```
auth       include	system-remote-login
account    include	system-remote-login
password   include	system-remote-login
session    include	system-remote-login
```
this part should be supported by distro maintainers, imo... upstream provide just an example.

**Comment by moobyfr** [comment]
At least, the pam file for debian jessie, which uses systemd is ok.

**Comment by metalefty** [comment]
Solved by 8799bbac42f0e4ba0f878b9c0c39229bbc4863fb.
